src/Tag/index.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cc from "classcat";
+
+const noop = () => {};
+
+/**
+ * A rounded rectangle inline label.
+ * The user has the option of firing a callback for 'dismissible' Tags.  
+ */
+const Tag = ({
+  kind = "subdued", // outline, subdued, x-tag (cactus400) #7fbc5b; #7FBC5B oneof
+  onDismiss = noop,
+  label = "my cool tag",
```

Comment:
might be good to make this an empty string by default - if we're missing data or something else makes the `label` undefined, this message could show up in prod

Reply:
Good call!